regolith/schemas.py

```diff
@@ -31,6 +31,13 @@
 SERVICE_TYPE = ["profession", "university", "school", "department"]
 TODO_STATI = ["started", "finished", "cancelled", "paused"]
 PROJECTA_STATUS = ["proposed", "started", "converged", "finished"]
+# kickoff, deliverable, milestones, and the projectum all have a status field, want different options for each?
+PROJECTUM_STATUS = ["proposed", "started", "converged", "finished", "cancelled", "paused"]
+# this is every milestone type I found in the projecta db
+MILESTONE_TYPE = ["pr", "meeting", "other", "paper", "pull request", "release", "email", "handin",
```

Comment:
nice work!

pr stands for pull request, so please remove pull request (we will have to update the database to make it validate!)

let's also remove (and we will need a PR on the database to correct these)
code (covered by release or pr)
task (not a deliverable)
file (covered by handin?)
finalized_doc (covered by approval)
software (covered by pr etc.)
merged_pr (covered by pr)
finalized_list (covered by approval...see below)
manuscript (covered by paper)
replace approved_file with approval